packages/editor/src/components/block-manager/index.js

```diff
@@ -2,69 +2,49 @@
  * WordPress dependencies
  */
 import { store as blocksStore } from '@wordpress/blocks';
-import { useDispatch, useSelect } from '@wordpress/data';
+import { useSelect } from '@wordpress/data';
 import { SearchControl, Button } from '@wordpress/components';
 import { __, _n, sprintf } from '@wordpress/i18n';
 import { useEffect, useState } from '@wordpress/element';
 import { useDebounce } from '@wordpress/compose';
 import { speak } from '@wordpress/a11y';
-import { store as preferencesStore } from '@wordpress/preferences';
 
 /**
  * Internal dependencies
  */
-import { unlock } from '../../lock-unlock';
-import { store as editorStore } from '../../store';
 import BlockManagerCategory from './category';
 
-export default function BlockManager() {
+/**
+ * Provides a list of blocks with checkboxes.
+ *
+ * @param {Object}   props                    Props.
+ * @param {Array}    props.blockTypes         An array of blocks.
+ * @param {Array}    props.selectedBlockTypes An array of selected blocks.
+ * @param {Function} props.onChange           Function to be called when the selected blocks change.
+ */
+export default function BlockManager( {
+	blockTypes,
+	selectedBlockTypes,
```

Comment:
I can't see any errors so it might be a small thing, but I'm wondering that, if this is going to be a shared component, do you think any data validation is appropriate?

If `selectedBlockTypes` isn't an array, a bunch of stuff would break.

Reply:
For now, I plan to release this component as a private component, so there shouldn't be any problems as long as it's used correctly within Gutenberg.

However, it would be better to at least have a conversion to an array, so I dealt with it in 242ba0e.

Reply:
I see, let's remove this validation for now.